README.md

```diff
@@ -161,9 +161,13 @@ tools:
     format-stdin: true
 
   javascript-eslint: &javascript-eslint
-    lint-command: 'eslint -f unix --stdin'
+    lint-command: 'eslint -f visualstudio --stdin --stdin-filename ${INPUT}'
```

Comment:
As t his configuration mentions windows it's possible `--stdin-filename ${INPUT}` isn't needed, not sure honestly as I'm on a unix system but it's needed for my configuration to work.